src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  column-gap: 48px;
+  row-gap: 46px;
+  padding: 50px 15px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  &__item {
+    box-sizing: border-box;
+    width: 200px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: #fff;
+    padding: 0 16px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    transition: all 0.3s ease 0s;
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
+
+  &__image {
+    padding-top: 32px;
```

Comment:
it is better to add internal geometry to the container. In your case it's `card__item`